src/content/measure.ts

```diff
@@ -97,15 +97,15 @@ export function extractMeasureMetadata(text: string): MeasureMeta | undefined {
 }
 
 export type ConvertedMeasure = {
-  unit: '帖' | '畳' | 'm2';
+  unit: '帖' | '畳' | 'm2' | 'sq ft';
   value: number;
   alt?: Array<AlternateMeasure>;
 };
 
 export type AlternateMeasure = {
   type: 'kyouma' | 'chuukyouma' | 'edoma' | 'danchima';
   label?: string;
-  unit: '畳' | 'm2';
+  unit: '畳' | 'm2' | 'sq ft';
```

Comment:
Just a thought, in the Japanese localization, do we still want to display "sq ft" or "平方フィート" or even ft²?

Reply:
This crossed my mind, but I think this is Americentric enough that displaying it in the Latin alphabet is reasonable even in Japanese. If I'm not mistaken, the Japanese localization still displays m², right, rather than 平方メートル? So I think ft², for consistency.